static/src/javascripts/projects/common/modules/commercial/hosted-gallery.js

```diff
@@ -0,0 +1,39 @@
+define([
+    'bean',
+    'lodash/functions/debounce',
+    'bonzo',
+    'fastdom',
+    'common/utils/$'
+], function (
+    bean,
+    debounce,
+    bonzo,
+    fastdom,
+    $
+) {
+
+
+    function init() {
+        var $scrollEl = $('.hosted-gallery__scroll-container');
+        var $images = $('.hosted-gallery__image');
+        if (!$scrollEl.length) {
+            return;
+        }
+
+        bean.on($scrollEl[0], 'scroll', debounce(function (e) {
```

Comment:
Specify the number of milliseconds to delay in `debounce` so the function won't be fired that often and performance will be improved.